single_cell/workflows/pseudo_bulk_qc/scripts/single_cell_qc_plots.py

```diff
@@ -305,19 +306,26 @@ def load_allele_data(haplotype_allele_data):
 def load_allele_data(haplotype_allele_data):
     allele_data = []
 
-    for chunk in csvutils.CsvInput(haplotype_allele_data).read_csv(chunksize=1e6):
+    unique_cells = []
+    #csvutils is breaking - haplotype allele data file has no yaml
+    for chunk in pd.read_csv(haplotype_allele_data, sep="\t", chunksize=1e6):
```

Comment:
the pipeline should generate a yaml. 

just assume that the yaml is there. 


for the test dataset: lets generate a yaml.